guidelines/SAMPLE-MAINTAINERS.md

```diff
@@ -0,0 +1,112 @@
+  an interest in becoming a maintainer for the repository.
+- A PR is created to update this file to add the proposed maintainer to the list of active maintainers.
+- The PR is authored by an existing maintainer or has a comment on the PR from an existing maintainer supporting the proposal.
+- The PR is authored by the proposed maintainer or has a comment on the PR from the proposed maintainer expressing their interest in being a maintainer.
+  - The PR or comment from the proposed maintainer must include their
+    willingness to be a long-term (more than 6 month) maintainer.
+- Once the PR and necessary comments have been received, an approval timeframe begins.
+- The PR **MUST** be communicated on all appropriate communication channels, including relevant community calls, chat channels and mailing lists. Comments of support from the community are welcome.
+- An add maintainer PR may be vetoed by another maintainer prior to merging.
+  - Vetoes must be accompanied by a public explanation as a comment in the
+    PR.
+  - The explanation of the veto must be reasonable.
+  - A veto can be retracted. In that case the approval/veto timeframe is reset.
+  - It is bad form to veto, retract, and veto again.
+- The PR is merged and the proposed maintainer becomes a maintainer if
+  - No maintainer posts a veto AND
+    - Either: Two weeks have passed since at least 3 Maintainer PR approvals has been recorded, OR
+    - An absolute majority of maintainers have approved the PR.
+- Once the add maintainer PR has been merged, any necessary updates to the GitHub Teams are made.
+
+## Removing Maintainers
+
+Being a maintainer is not a status symbol or a title to be carried
+indefinitely. It will occasionally be necessary and appropriate to move a
+maintainer to emeritus status. This can occur in the following situations:
+
+- Resignation of a maintainer.
+- Violation of the Code of Conduct warranting removal.
+- Inactivity.
+  - A general measure of inactivity will be no commits or code review comments
+    for one reporting quarter. This will not be strictly enforced if
+    the maintainer expresses a reasonable intent to continue contributing.
+  - Reasonable exceptions to inactivity will be granted for known long term
+    leave such as parental leave and medical leave.
+- Other circumstances at the discretion of the other Maintainers.
+
+The process to move a maintainer from active to emeritus status is comparable to the process for adding a maintainer, outlined above. In the case of voluntary
+resignation, the Pull Request can be merged following a maintainer PR approval. If the removal is for any other reason, the following steps **SHOULD** be followed:
+
+- A PR is created to update this file to move the maintainer to the list of emeritus maintainers.
+- The PR is authored by an existing maintainer or has a comment on the PR from an existing maintainer supporting the proposal.
```

Comment:
Note - often times no existing maintainer wants to retire a colleague, it has been helpful to be able to ask a Hyperledger staff member to open a retirement PR. In fact, https://github.com/hyperledger/toc/blob/gh-pages/governing-documents/inactivity.md mentions it would be a TOC member or Hyperledger staff member, but I think it is acceptable for any project maintainer to do it.

Reply:
Having done this many times by request, I agree with @denyeart

Reply:
I think adding “or a Hyperledger Organization GitHub Administrator” would make sense.  I’ll try that.